internal/fs/inode/dir.go

```diff
@@ -147,6 +147,10 @@ type DirInode interface {
 	// should be invalidated or not.
 	ShouldInvalidateKernelListCache(ttl time.Duration) bool
 
+	//InvalidateKernelListCache sets the prevDirListingTimeStamp to zero so that
```

Comment:
Let's not use `prevDirListingTimeStamp` as these are internal details. We can say something like - 
`InvalidateKernelListCache guarantees that the subsequent list call will be served from GCSFuse.`